### PR TITLE
Clickable mail link for "contact support" messages

### PR DESCRIPTION
**Pull request by otterbotter**
https://trello.com/c/Y6QmPWMj/264-contact-support-notices-as-a-user-i-would-like-to-know-how-to-contact-support

### CODE REVIEW

File: src/js/components/account/user-registration.js [hunk 1]
```diff
@@ -150,12 +150,12 @@ export class UserRegistration {
       })
       .fail((jqXHR, textStatus) => {
         try {
-          $fail.find("div").text(formatError(jqXHR));
-          $fail.show();
+          $fail.find("div").html(formatError(jqXHR));
+          $fail.showeew();
```
Comment (schalkneethling) [inline src/js/components/account/user-registration.js:154]: intentional?

File: src/js/components/account/forgot-password.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ export class ForgotPassword {
           .empty()
           .append(
             "Something went wrong while communicating with the server. " +
-              "Please try again or contact support."
+              "Please try again or <a href='mailto:cape-agulhas-app@openup.org.za'>contact support</a>."
```
Comment (schalkneethling) [inline src/js/components/account/forgot-password.js:80]: I am wondering. We have this constants file: https://github.com/OpenUpSA/muni-portal-frontend/blob/master/src/js/components/constants.js

Perhaps we should add something like:

```javascript
export const SUPPORT_EMAIL = "cape-agulhas-app@openup.org.za";
```

Then we can do something like:

```javascript
import { SUPPORT_EMAIL } from ""../constants";

...

`"Please try again or <a href="mailto:${SUPPORT_EMAIL}">contact support</a>.`
```

Should we need to change the email address in the future, we then only need to do it in one place.

**Review by schalkneethling — COMMENTED** [review]
Two items to have a look at. One may just be a typo. Thank you, @otterbotter

File: src/js/components/account/forgot-password.js (hunk 2 above)
Comment (otterbotter) [reply src/js/components/account/forgot-password.js:80]: Yeah I agree; I did think about doing the same, but we were already hard coding this in many places so I left it.

I think it is worthwhile; I'll do the refactoring in this PR. We'll just need to make sure we test it properly so nothing breaks.

File: src/js/components/account/user-registration.js (hunk 1 above)
Comment (otterbotter) [reply src/js/components/account/user-registration.js:154]: Woops! Thanks

**Pull request merged**